hashicorp/boundary-ui
# feat: 🎸 Allow auto connect to SSH targets on windows

## Description
We now check if the user has an SSH client on their path which lets us auto connect to an SSH target on windows. I used the package [which](https://www.npmjs.com/package/which) because it doesn't use a `child_process` spawn as many other packages do and just checks the user's path. It works well on windows as you don't have to specifiy `.exe` as compared to other packages and it only has 1 dependency.

## How to Test
I produced some builds [here](https://github.com/hashicorp/boundary-ui-releases/actions/runs/6674102583) that has the changes.

On windows, you can confirm this works by testing that the auto connect works with an SSH target now. You can then remove the openSSH binaries from your path and restart electron and confirm now it won't try to auto connect.
 
## Checklist:
